### DISCUSSION TITLE
how to add additional info when i use insertImage api as i showed below

### DISCUSSION THREAD

**Discussion by Just-focus** [remirror/remirror#1401] [Q&A] [answered]
how to add additional info，thanks

![image](https://user-images.githubusercontent.com/18374969/144413403-9ad18473-5366-43d9-9086-2ac2f5620a4d.png)

![image](https://user-images.githubusercontent.com/18374969/144413526-e805ab4f-1bca-4133-aba0-e0651a66dbc9.png)

**Comment by ocavue** [comment, marked as answer]
Hi @Just-focus. I hope the example in #1416 answers your question. In this example, you can insert an image with `figcaptionText` attribute by adding `figcaptionText` to the first argument of `insertImage` command.

**Reply by Just-focus** [reply]
Thanks！i use the ReactComponent api to solve this problem
